grammars/coffeescript.cson

```diff
@@ -105,7 +105,7 @@
     'name': 'string.quoted.script.coffee'
     'patterns': [
       {
-        'match': '(x\h{2}|[0-2][0-7]{0,2}|3[0-6][0-7]|37[0-7]?|[4-7][0-7]?|.)'
+        'match': '(x[0-9A-Fa-f]{2}|[0-2][0-7]{0,2}|3[0-6][0-7]|37[0-7]?|[4-7][0-7]?|.)'
```

Comment:
Looks like this is missing a leading `(\\\\)`?  I'm pretty sure it should look identical to the hunk below.